LibCarla/source/carla/trafficmanager/LocalizationUtils.cpp

```diff
@@ -232,6 +233,43 @@ namespace LocalizationConstants {
     }
 
   }
+  SimpleWaypointPtr TrackTraffic::GetTargetWaypoint(Buffer& waypoint_buffer,const float& target_point_distance,uint64_t& index)
```

Comment:
Receive the parameter target_point_distance as a copy instead of reference as it will avoid unnecessary local variable creation in the caller logic for a simple primitive type.

Reply:
As discussed . This code will be same . So this comment will not be implemented